cobalt/akn.py

```diff
@@ -178,11 +263,23 @@ def meta(self):
     @property
     def title(self):
         """ Short title """
-        return self.meta.identification.FRBRWork.FRBRalias.get('value')
+        # look for the FRBRalias element with name="title", falling back to any alias
+        title = None
+        for alias in self.meta.identification.FRBRWork.iterchildren(f'{{{self.namespace}}}FRBRalias'):
```

Comment:
Am I understanding correctly that you're not really expecting there to be more than one `alias`?
In which case, if you're using `value` anyway, why bother checking for the name "title"?
But if you were expecting several `alias`es, would it make more sense to loop over them all first, looking for the name "title" and use that one's value, and only after that fall back to the first `value` you find? or have I got the wrong end of the stick

Reply:
We assume there are multiple. The first one named title that we find, we will use. Otherwise, we simply use the last alias we found.

Reply:
We have to loop over all of them, because the last one might be named title.

Reply:
I see how it works now! it only returns `title` at the very end. miscuzi.